cmd/operator/main.go

```diff
@@ -72,7 +72,7 @@ func init() {
 	flag.IntVar(&chaosLevel, "chaos-level", -1, "DO NOT USE IN PRODUCTION - level of chaos injected into the etcd clusters created by the operator.")
 	flag.BoolVar(&printVersion, "version", false, "Show version and quit")
 	// create-crd will be removed once the default behavior is not to create a CRD
```

Comment:
is this comment still accurate?
`// create-crd will be removed once the default behavior is not to create a CRD`

Reply:
Ah, good catch. No this flag will now stay since the default is creating a crd.